index.html

```diff
@@ -15,24 +15,17 @@
     <script src="https://cdn.datatables.net/1.10.22/js/dataTables.bootstrap4.min.js"></script>
     <script src="https://cdnjs.cloudflare.com/ajax/libs/moment.js/2.29.0/moment.min.js"></script>
     <style>
-        body {
-            padding-top: 2rem;
-            padding-bottom: 2rem;
-        }
-        .black {
-            color: #292b2c;
-        }
     </style>
```

Comment:
lets remove the empty `<style>` tag since its empty now.

Reply:
It's removed now.